src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/ExcelAdaptorServiceImpl.java

```diff
@@ -105,22 +115,36 @@ private void validateHeaders(List<String> headersToBeValidated,
         Field rowField = getRowIdField((Class<Object>) classType);
         while (rowIterator.hasNext()) {
             Row row = rowIterator.next();
-            //Skipping empty rows
-            if (checkIfRowIsEmpty(row)) {
-                continue;
+            Object bean = null;
+            try {
+                if (checkIfRowIsEmpty(row)) {
+                    continue; //Skipping empty rows
```

Comment:
why we are doing this? if there is an empty row in the file then we should consider it as end of the file and exit from loop

Reply:
what about the row is in between 2 valid rows ?